src/broker/cas_function.c

```diff
@@ -2742,17 +2747,17 @@ update_error_query_count (T_APPL_SERVER_INFO * as_info_p, const T_ERROR_INFO * e
     }
 }
 
-static char
+#if !defined (CAS_FOR_CGW)
+static void
 logddl_is_exist_ddl_stmt (T_SRV_HANDLE * srv_handle)
 {
-  char stmt_type = -1;
   for (int i = 0; i < srv_handle->num_q_result; i++)
     {
       if (logddl_is_ddl_type (srv_handle->q_result[i].stmt_type) == true)
 	{
-	  stmt_type = srv_handle->q_result[i].stmt_type;
-	  break;
+	  logddl_set_stmt_type (srv_handle->q_result[i].stmt_type);
+	  return;
```

Comment:
the function name does not explain the routine. how about renaming the function?
```
logddl_check_ddl ()...
```

Reply:
Changed to logddl_check_have_ddl_stmt().
Thank you.